components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.m

```diff
@@ -80,10 +86,20 @@ - (void)setupAppBar {
 
   self.appBar.headerViewController.headerView.tintColor = [UIColor whiteColor];
   self.appBar.headerViewController.headerView.minimumHeight = 76 + 72;
+   
+   UIFont *font;
+   if (isRunningiOS9OrGreater()) {
+      font = [UIFont monospacedDigitSystemFontOfSize:14 weight:UIFontWeightRegular];
+   } else {
+      UIFont* font = [UIFont systemFontOfSize:14];
```

Comment:
This crashes on IOS 8.3 because the `UIFont *` declared inside the `else` shadows the outer `font` variable, and because the descriptor is `nil`. It should not crash if you change to:

```objectivec
UIFont *font;
if (isRunningiOS9OrGreater()) {
  font = [UIFont monospacedDigitSystemFontOfSize:14 weight:UIFontWeightRegular];
} else {
  font = [UIFont systemFontOfSize:14];
  UIFontDescriptor *descriptor =
      [[font fontDescriptor] fontDescriptorWithSymbolicTraits:UIFontDescriptorTraitMonoSpace];
  if (descriptor) {
    font = [UIFont fontWithDescriptor:descriptor size:0.0];
  }
}
```